resources/ingredients.json

```diff
@@ -482,158 +989,85 @@
       "Water Issue": 0,
       "CO2 Issue": 0,
       "Ethical Alternatives": [
-          ""
-      ],
-      "Animal Product": 0
-  },
-  "wholeWheatTortillas": {
-      "Display Name": "Whole Wheat Tortillas",
-      "Water Issue": 0,
-      "CO2 Issue": 0,
-      "Ethical Alternatives": [
-          ""
+          "wholeWheatBread"
       ],
       "Animal Product": 0
   },
-  "yogurt": {
-      "Display Name": "Yogurt",
-      "Water Issue": 1,
-      "CO2 Issue": 1,
-      "Ethical Alternatives": [
-          "almondMilk"
-      ],
-      "Animal Product": 1
-  },
-  "grilledChicken": {
-      "Display Name": "Grilled Chicken",
-      "Water Issue": 0,
-      "CO2 Issue": 1,
-      "Animal Product": 1,
-      "Ethical Alternatives": [
-          "blackBeanBurger",
-          "refriedBeans"
-      ],
-      "Categories": [
-          "Lunch",
-          "Dinner",
-          "Meat"
-      ]
-  },
-  "sourCream": {
-      "Display Name": "Sour Cream",
-      "Water Issue": 1,
-      "CO2 Issue": 1,
-      "Animal Product": 1,
-      "Ethical Alternatives": [
-          "veganCheese"
-      ],
-      "Categories": [
-          "Breakfat",
-          "Lunch",
-          "Dinner",
-          "Dairy"
-      ]
-  },
-  "garlic": {
-      "Display Name": "Garlic",
-      "Water Issue": 0,
+  "whiteChocolateChips": {
+    "Display Name": "White Chocolate Chips",
+    "Water Issue":0,
+    "CO2 Issue": 0,
+    "Ethical Alternatives": [
+        ""
+    ],
+    "Animal Product": 0
+},
+
+  "wholeWheatBread": {
+    "Display Name": "Whole Wheat Bread",
+    "Water Issue": 1,
+    "CO2 Issue": 0,
+    "Ethical Alternatives": [
+        ""
+    ],
+    "Animal Product": 0
+},
+
+"wholeWheatBun": {
+    "Display Name": "Whole Wheat Bun",
+    "Water Issue": 0,
+    "CO2 Issue": 0,
+    "Ethical Alternatives": [
+        ""
+    ],
+    "Animal Product": 0
+},
+   "wholeWheatFlour": {
+      "Display Name": "Whole Wheat Flour",
+      "Water  Issue": 1,
       "CO2 Issue": 0,
-      "Animal Product": 0,
-      "Ethical Alternatives": [
-          ""
-      ],
-      "Categories": [
-          "Breakfat",
-          "Lunch",
-          "Dinner",
-          "Spices"
-      ]
-  },
-  "steak": {
-      "Display Name": "Steak",
-      "Water Issue": 1,
-      "CO2 Issue": 1,
-      "Animal Product": 1,
       "Ethical Alternatives": [
-          ""
+        "quinoaFlour"
       ],
-      "Categories": [
-          "Dinner"
-      ]
+    "Animal Product": 0
   },
-  "blackBeanBurger": {
-      "Display Name": "Black Bean Burger",
-      "Water Issue": 0,
+    "wholeWheatLasagna": {
+      "Display Name": "Whole Wheat Lasagna",
+      "Water  Issue": 1,
```

Comment:
Looks like there may be an extra space in "Water Issue" here that would cause issues with the key lookup